lib/app/endpoint/home.rb

```diff
@@ -385,6 +399,18 @@ class Home < Endpoint
           end
           client.default_json
           submitted_test_cases = params[:test_case].split(',')
+
+          instance.reload # ensure that we have all the latest data
+
+          total_tests = submitted_test_cases.reduce(0) do |total, set|
+            sequence_test_count =
+              test_set
+                .test_case_by_id(set)
+                .sequence.new(instance, client, settings.disable_tls_tests)
```

Comment:
I don't think you need to instantiate sequence here... `test_count` should work on the class as well.  Did you try that and it didn't work?

Reply:
You're right, didn't realize that wasn't needed. All updated now, thanks!